src/unk_0206B910.c

```diff
@@ -147,8 +147,8 @@ static BOOL sub_0206B984(TaskManager *taskManager) {
     case 4: {
         PartyMenuArgs *partyMenu = r7->partyMenu;
         u32 index = r7->unk04->unk30 + 1;
-        if (partyMenu->unk_26 != 7) {
-            Pokemon *mon = r7->unk04->mons[partyMenu->unk_26];
+        if (partyMenu->partySlot != 7) {
```

Comment:
prolly need a const here for MAX_PARTY_SIZE then say partySlot == MAX_PARTY_SIZE + 1

Reply:
I think we already have one somewhere

Reply:
i'd rather have the constant describe what's actually going on here, which is beyond my ability to suss out because apparently there's two sets of constants and the meaning of 6 and 7 is flipped between them?